### PR TITLE
Minor revisions to novel vaccines

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

We thank the reviewer for their kind words and helpful suggestions. We have made the following changes to the manuscript:

3) Line 239 - This reference is not available

This work in progress is currently hosted on the GitHub pages rendering of the broader COVID-19 Review Consortium project. However, we suspect many readers may encounter a lag when accessing the bookmarked link due to the large amount of material on this page. We are working on identifying a solution to this problem, but in the meantime, we will update the reference to refer to the COVID-19 Review Consortium project itself, which includes this manuscript.

4) Line 276 - Malaria is not a bacterial disease

Thank you for catching this error. This line has been corrected to identify malaria as a parasitic disease.

**5) Line 683 - 2,686 represents 68% of the participants in the trial **
Thank you for catching this unclear sentence. It has been rephrased to state: Among these participants, 3,179 (80%) had received at least one dose of an mRNA vaccine, and of those, 2,686(84%) were fully vaccinated, corresponding to 68% of trial participants overall.

6) Line 266 and 287 - Which immune stimulant molecules? Which two vaccines?

Line 266 has been revised to state: "Plasmids can also be designed to act as adjuvants by encoding molecules such as cytokines that supplement the immune response [@doi:10.1358/dot.2018.54.5.2807864]."
Line 287 erroneously identified the number of veterinary plasmid-vectored DNA vaccines. This line has been modified to read: Prior to COVID-19, however, plasmid-vectored DNA vaccines had been approved for commercial use only in veterinary populations [@doi:10.1016/B978-0-323-35761-6.00067-5; @doi:10.5772/intechopen.76754; @doi:10.1016/j.drudis.2021.06.008]. Between 2005 and 2006, several DNA vaccines were developed for non-human animal populations, including against viruses including a rhabdovirus in fish [@doi:10.3354/dao064013], porcine reproductive and respiratory syndrome virus [@doi:10.2527/2006.843733x], and West Nile virus in horses [@doi:10.2460/javma.2005.226.240]. Within the past five years, additional plasmid-vectored vaccines for immunization against viruses were developed against a herpesvirus (in mice) [@doi:10.1089/vim.2020.0082] and an alphavirus (in fish) [@doi:10.1016/j.fsi.2018.07.012].

7) The reference list does not follow the format required by mSystems. In particular, volume and page # are missing.

The change has been made.

**Note: the 8th reviewer comment needs to be handled in a separte PR that I will link here!**

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/24.vaccines-novel.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ Table: DNA vaccines approved in at least one country [@url:https://covid19.track
 #### Plasmid-Vectored DNA Vaccines
 
 Many DNA vaccines use a plasmid vector-based approach, where the sequence encoding the antigen(s) against which an immune response is sought are cultivated in a plasmid and delivered directly to an appropriate tissue [@url:https://www.who.int/biologicals/areas/vaccines/dna/en].
-Plasmids can also be designed to act as adjuvants by encoding molecules that supplement the immune response, such as immune stimulant molecules [@doi:10.1358/dot.2018.54.5.2807864].
+Plasmids can also be designed to act as adjuvants by encoding molecules such as cytokines that supplement the immune response [@doi:10.1358/dot.2018.54.5.2807864].
```
Comment (rando2) [inline content/24.vaccines-novel.md:162]: @RLordan curious what you think here, the reviewer asked "Which immune stimulant molecules?"
The reference mentions cytokines "and immune stimulant molecules" but does not elaborate.

**Review by rando2 — COMMENTED** [review]
Hey @RLordan, I had one question for you! My other major question is just if anything big has changed in the past couple weeks or so that we should update. I feel like I've lost all sense of time... I believe we did update for the bivalent booster.

**Comment by rando2** [comment]
Reference format:

**Review by RLordan — APPROVED** [review]

**Comment by RLordan** [comment]
> Hey @RLordan, I had one question for you! My other major question is just if anything big has changed in the past couple weeks or so that we should update. I feel like I've lost all sense of time... I believe we did update for the bivalent booster.

Hey @rando2, I don't think so, the bivalent vaccine is the main thing. There is always interesting studies coming out, but in terms of ground breaking stuff - I don't think so... here is a few tidbits

A few myocarditis/vaccines studies out, benefits still outweigh risks - https://www.acpjournals.org/doi/10.7326/M22-2274
Novavax induces strong t cell response - https://www.jci.org/articles/view/160898
4th vaccine protects against BA.5 - https://doi.org/10.1101/2022.09.30.22280573

File: content/24.vaccines-novel.md (hunk 1 above)
Comment (rando2) [reply content/24.vaccines-novel.md:162]:
```suggestion
Plasmids can also be designed to act as adjuvants by targeting essential regulators of pathways such as the inflammasome or simply just specific cytokines [@doi:10.1358/dot.2018.54.5.2807864; @doi:[10.1371/journal.pone.0231138](https://doi.org/10.1371%2Fjournal.pone.0231138)].
```
Awesome! I propose changing it to your wording!

**Comment by rando2** [comment]
Thank you @RLordan! I just added a couple sentences covering those new papers.

**Comment by RLordan** [comment]
> Thank you @RLordan! I just added a couple sentences covering those new papers.

Thank you!

**Pull request merged**